src/modules/mavlink/mavlink_main.cpp

```diff
@@ -2034,7 +2086,28 @@ Mavlink::task_main(int argc, char *argv[])
 				err_flag = true;
 			}
 
+			break;
+
+//multicast
+#if defined(CONFIG_NET_IGMP) && defined(CONFIG_NET_ROUTE)
```

Comment:
This is NuttX specific but probably also works on Linux. So probably best to do under one define (as @davids5 suggested).